### PR TITLE
Team members

### PR DESCRIPTION
**Pull request by Barterio**
What: 

Why:

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

File: satellite/console/consoleweb/consoleql/projectmember.go [hunk 1]
```diff
@@ -38,8 +38,73 @@ func graphqlProjectMember(service *console.Service, types *TypeCreator) *graphql
 	})
 }
 
+func graphqlProjectMembersCursor() *graphql.InputObject {
```
Comment (rikysya) [inline satellite/console/consoleweb/consoleql/projectmember.go]: Add comments for consistency

File: satellite/console/consoleweb/consoleql/projectmember.go [hunk 2]
```diff
@@ -38,8 +38,73 @@ func graphqlProjectMember(service *console.Service, types *TypeCreator) *graphql
...
+				Type: graphql.NewNonNull(graphql.Int),
+			},
+			OrderArg: &graphql.InputObjectFieldConfig{
+				Type: graphql.NewNonNull(graphql.Int),
+			},
+		},
+	})
+}
+
+func graphqlProjectMembersPage(types *TypeCreator) *graphql.Object {
```
Comment (rikysya) [inline satellite/console/consoleweb/consoleql/projectmember.go]: Add comments for consistency

File: satellite/console/consoleweb/consoleql/projectmember.go [hunk 3]
```diff
@@ -38,8 +38,73 @@ func graphqlProjectMember(service *console.Service, types *TypeCreator) *graphql
...
+	})
+}
+
 // projectMember encapsulates User and joinedAt
 type projectMember struct {
 	User     *console.User
 	JoinedAt time.Time
 }
+
+type projectMembersPage struct {
```
Comment (rikysya) [inline satellite/console/consoleweb/consoleql/projectmember.go]: Add comments for consistency

File: satellite/console/consoleweb/consoleql/typecreator.go [hunk 4]
```diff
@@ -18,18 +18,20 @@ type TypeCreator struct {
...
-	userInput         *graphql.InputObject
-	projectInput      *graphql.InputObject
-	bucketUsageCursor *graphql.InputObject
+	user              *graphql.Object
+	project           *graphql.Object
+	projectUsage      *graphql.Object
+	bucketUsage       *graphql.Object
+	bucketUsagePage   *graphql.Object
+	projectMember     *graphql.Object
+	projectMemberPage *graphql.Object
```
Comment (rikysya) [inline satellite/console/consoleweb/consoleql/typecreator.go]: projectMembersPage?

File: satellite/console/projectmembers.go [hunk 5]
```diff
@@ -14,8 +14,8 @@ import (
 type ProjectMembers interface {
 	// GetByMemberID is a method for querying project members from the database by memberID.
 	GetByMemberID(ctx context.Context, memberID uuid.UUID) ([]ProjectMember, error)
-	// GetByProjectID is a method for querying project members from the database by projectID, offset and limit.
-	GetByProjectID(ctx context.Context, projectID uuid.UUID, pagination Pagination) ([]ProjectMember, error)
+	// GetByProjectIDTotal is a method for querying project members from the database by projectID and cursor.
+	GetByProjectIDTotal(ctx context.Context, projectID uuid.UUID, cursor ProjectMembersCursor) (*ProjectMembersPage, error)
```
Comment (rikysya) [inline satellite/console/projectmembers.go]: What `total` suffix stands for?

File: web/satellite/src/api/projectMembers.ts [hunk 6]
```diff
@@ -70,37 +71,53 @@ export async function deleteProjectMembersRequest(projectID: string, emails: str
...
                 }
-            }`
+            `
             ),
             fetchPolicy: 'no-cache',
             errorPolicy: 'all',
         }
     );
 
+    console.log(response)
```
Comment (rikysya) [inline web/satellite/src/api/projectMembers.ts]: Remove debug code

File: web/satellite/src/api/projects.ts [hunk 7]
```diff
@@ -64,6 +65,7 @@ export async function fetchProjectsRequest(): Promise<RequestResponse<Project[]>
         }
     );
 
+    console.log(response)
```
Comment (rikysya) [inline web/satellite/src/api/projects.ts]: Remove debug code

File: satellite/console/consoleweb/consoleql/project.go [hunk 8]
```diff
@@ -325,3 +319,16 @@ func fromMapBucketUsageCursor(args map[string]interface{}) (cursor console.Bucke
 	cursor.Search, _ = args[SearchArg].(string)
 	return
 }
+
+func fromMapProjectMembersCursor(args map[string]interface{}) (cursor console.ProjectMembersCursor) {
```
Comment (ifraixedes) [inline satellite/console/consoleweb/consoleql/project.go]: Don't use named return parameters only for the purpose of declaring the variable (https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters)

File: satellite/satellitedb/projectmembers.go [hunk 9]
```diff
@@ -20,42 +20,73 @@ type projectMembers struct {
...
+		return nil, errs.New("page is out of range")
+	}
 	// TODO: LIKE is case-sensitive postgres, however this should be case-insensitive and possibly allow typos
 	reboundQuery := pm.db.Rebind(`
 		SELECT pm.*
 			FROM project_members pm
 				INNER JOIN users u ON pm.member_id = u.id
 					WHERE pm.project_id = ?
-					AND ( u.email LIKE ? OR
+					AND ( u.email LIKE ? OR
```
Comment (ifraixedes) [inline satellite/satellitedb/projectmembers.go]: Remove this change, it's just a space at the end of the line, before break line character.

File: web/satellite/src/components/team/footerArea/TeamMemberPaginationArea.vue [hunk 10]
```diff
@@ -0,0 +1,179 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+	<div>
+		<div class="pagination-container">
```
Comment (VitaliiShpital) [inline web/satellite/src/components/team/footerArea/TeamMemberPaginationArea.vue]: seems like more than 4 spaces here

File: web/satellite/src/components/team/footerArea/TeamMemberPaginationArea.vue [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+                return currentPage === pageCount;
+            }
+        }
+    })
+
+    export default class TeamMemberPaginationArea extends Vue {}
+</script>
+
+<style scoped lang="scss">
+	.pagination-container {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/team/footerArea/TeamMemberPaginationArea.vue]: seems like more than 4 spaces here

**Pull request closed without merging**